Algorithms/DataCompression/huffman.cs

```diff
@@ -27,184 +28,298 @@ public Huff(Huff leftChild, Huff rightChild)
 
     internal class Man
```

Comment:
Why this class is called Man and what is is stored in its properties Codec and Data?

Reply:
It's where the encoded text message gets stored temporarily, the name can be changed, is not relevant.